com.gitfiend.GitFiend.metainfo.xml

```diff
@@ -0,0 +1,73 @@
+      <li>Tags</li>
+      <li>Remotes: Add, remove or select which remote to pull and push to</li>
+      <li>Resolve merge conflicts: Select specific files, hunks or lines to include. Lines can be
+        rearranged</li>
+      <li>Undo: Undo un-pushed commits. Revert commits</li>
+      <li>Cherry-pick</li>
+      <li>Filter commits: Filter commits by branch or user</li>
+      <li>History for a specific file: Apply a file filter and navigate through all commits with
+        changes for that file</li>
+      <li>Search: Quickly search through history for code, commits, messages, users and branches.</li>
+      <li>Git submodules/multi-repo: Preview feature. Open a folder of repos, see the status of each
+        and run bulk actions such as fetch/pull/push and create branch.</li>
+    </ul>
+  </description>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <content_rating type="oars-1.1" />
+  <categories>
+    <category>Development</category>
+    <category>RevisionControl</category>
+  </categories>
+  <url type="homepage">https://gitfiend.com</url>
+  <url type="bugtracker">https://github.com/GitFiend/Support/issues</url>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Main screen</caption>
+      <image>https://gitfiend.com/resources/home-lin.png</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="0.39.4" date="2023-02-10" />
+    <release version="0.38.3" date="2022-10-19" />
+    <release version="0.37.2" date="2022-10-01" />
+    <release version="0.36" date="2022-08-27" />
+    <release version="0.35.3" date="2022-07-31" />
+    <release version="0.34.0" date="2022-05-13" />
+    <release version="0.33.0" date="2022-05-09" />
+    <release version="0.32.0" date="2022-01-30" />
+    <release version="0.31.0" date="2022-01-15" />
+    <release version="0.30.1" date="2021-12-16" />
+    <release version="0.29" date="2021-09-27" />
+    <release version="0.28" date="2021-07-25" />
+    <release version="0.27" date="2021-06-20" />
+    <release version="0.26" date="2021-03-20" />
+    <release version="0.25" date="2021-02-14" />
+    <release version="0.24" date="2020-07-05" />
+    <release version="0.23" date="2020-05-10" />
+    <release version="0.22.8" date="2020-03-15" />
+    <release version="0.22.6" date="2020-03-06" />
+    <release version="0.22" date="2020-03-05" />
+    <release version="0.21" date="2019-12-26" />
+    <release version="0.20" date="2019-10-13" />
+    <release version="0.19.5" date="2019-08-21" />
+    <release version="0.19.2" date="2019-07-20" />
+    <release version="0.19" date="2019-07-04" />
+    <release version="0.18" date="2019-05-26" />
+    <release version="0.17" date="2019-04-29" />
+  </releases>
+  <developer_name>Toby Suggate</developer_name>
+  <launchable type="desktop-id">com.gitfiend.gitfiend.desktop</launchable>
```

Comment:
this is case sensitive, that's why the build is failing.

Reply:
I was still figuring what the error was all about. appstreamcli didn't complain about this one though.

```
$ appstreamcli validate com.gitfiend.GitFiend.metainfo.xml
I: com.gitfiend.GitFiend:7: description-first-para-too-short
     Here's where GitFiends at feature-wise and what's coming up next.

? Validation was successful: infos: 1, pedantic: 1

```
Thanks again for the tip!

Reply:
because it doesn't. it's valid in the appstream.

but appstream compose expect to find that desktop file for the icons, which it doesn't because the case doesn't match.

Reply:
> but appstream compose expect to find that desktop file for the icons, which it doesn't because the case doesn't match.

A rather silly error on my side. Thanks for the heads up!